skglm/__init__.py

```diff
@@ -2,5 +2,6 @@
 
 from skglm.estimators import (  # noqa F401
     Lasso, WeightedLasso, ElasticNet, MCPRegression, MultiTaskLasso, LinearSVC,
-    SparseLogisticRegression, GeneralizedLinearEstimator, CoxEstimator
+    SparseLogisticRegression, GeneralizedLinearEstimator, CoxEstimator,
+    WeightedMCPRegression
```

Comment:
As far as I'm concerned I'd just expose `weights` in MCPRegression in order not to bloat the API

For WeightedLasso we went the other way to gain exposure, but here it's a more niche model so I reckon we can spare the estimator. WDYT?

Reply:
Done!